### DISCUSSION TITLE
Helping with translation

### DISCUSSION THREAD

**Discussion by PMorghK** [ppy/osu#16835] [Q&A] [answered]
any way that I could help with translation?

**Comment by HelloYeew** [comment, marked as answer]
You can help with translation in [osu-web crowdin](https://crowdin.com/project/osu-web). It has a resource on osu! website to but if you want to help on the client itself just get in client folder. (but some word in the lazer is using the word in web project btw)

**Reply by PMorghK** [reply]
Right, ty.
